### PR TITLE
Fix Shipping Simulator Price Table Number Format

### PR DESCRIPTION
**Pull request by juliomoreira**
Converts price value to decimal instead of cents to fix price display.

shipping-fix

### CODE REVIEW

File: react/components/ShippingSimulator/components/ShippingTableRow.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const ShippingTableRow = ({ name, shippingEstimate, price, intl }) => {
   } else if (price === 0) {
     valueText = intl.formatMessage({ id: 'shipping.free' })
   } else {
-    valueText = intl.formatNumber(price, { style: 'currency', currency: 'BRL' })
+    valueText = intl.formatNumber(price/100, { style: 'currency', currency: 'BRL' })
```
Comment (matheusps) [inline react/components/ShippingSimulator/components/ShippingTableRow.js]: I think this division should be optional.

Comment (juliomoreira) [reply react/components/ShippingSimulator/components/ShippingTableRow.js]: I don't see why.

The price value comes directly from the getShippingEstimates query which returns the value in cents, and it's totally fine, should be an standard if it's not already.

React-intl formatNumber method doesn't care about it and it's not part of his job to manipulate the values of attributes but it's expecting a decimal based number to be formatted.

So I don't see why we should control if the division is done or not, only if you're planing to change returned output of the query.

Comment (matheusps) [reply react/components/ShippingSimulator/components/ShippingTableRow.js]: Good point, it makes sense. Just add a CHANGELOG and your PR will be good to go, for me.

Comment (matheusps) [reply react/components/ShippingSimulator/components/ShippingTableRow.js]: Yeah, good argument. It makes sense. For me, just update the CHANGELOG file with your changes and the PR is good to go. 🙂

Comment (juliomoreira) [reply react/components/ShippingSimulator/components/ShippingTableRow.js]: I'll do that.

Sorry for not going through all the PR guidelines.

Comment (matheusps) [reply react/components/ShippingSimulator/components/ShippingTableRow.js]: No problem 🙂

**Pull request merged**